tests/test_kernelshap.py

```diff
@@ -53,7 +53,7 @@ def test_shap_explain_image(self):
         n_segments = 50
```

Comment:
n_segments is explicitly set to 50, and only used to test against. It doesn't make a lot of sense to overwrite this value, as proposed in this PR, before it is tested against. The proposed changes would test whether the output of the method (shap_values and segments) are consistent with each other, which is a good test by itself. However, I think that the idea of this test was to check the outcome of the method against some knowledge that we have prior to running the method, like some ground truth or regression test.
It's a shame it is not explained why this value of 50 is what is expected as an outcome exactly.  @geek-yang, git blame points at you ;-) (https://github.com/dianna-ai/dianna/commit/b8dd755236d2acf3ee3a49420f481402226ea4ac) Could you explain this value and/or review this PR?

Reply:
😅 To be honest Chris I forgot why I set it to be exactly 50. But I think it is a bug given that the number of labels can be slightly different, as `skimage.segmentation.slic` already says `n_segments` is actually "The (approximate) number of labels".

So it should be corrected as what @WillemSpek suggests. Thanks for spotting this @cwmeijer and @WillemSpek . Shame on me 😂

PS: Oh yes, the commit was made at the first week of 2022, right after new year. You guys were all on holiday and I'm the only one working at that moment. What a piece of painful memory 😭. But anyway...my bad 😂 .